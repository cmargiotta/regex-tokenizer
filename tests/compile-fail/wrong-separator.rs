use regex_tokenizer::tokenizer;

fn main() {
    tokenizer! {
        Test

        "a" => A
        "b" => B
        "c"
    }
}
