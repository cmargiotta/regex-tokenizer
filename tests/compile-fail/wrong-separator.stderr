error: Arrow => required
 --> tests/compile-fail/wrong-separator.rs:9:9
  |
9 |         "c"
  |         ^^^
